monkey/monkey_island/cc/models/agent.py

```diff
@@ -26,7 +27,7 @@ class Agent(MutableInfectionMonkeyBaseModel):
     parent_id: Optional[AgentID] = Field(allow_mutation=False)
     """The ID of the parent agent that spawned this agent"""
 
-    cc_server: str = Field(default="")
+    cc_server: Optional[SocketAddress]
```

Comment:
Optional means something or None, not something or empty string, though I doubt it's used anywhere

Reply:
I couldn't find any usages of cc_server where an empty string was required.